src/idea.js

```diff
@@ -1,8 +1,8 @@
 class Idea {
-  constructor(title, body, star) {
+  constructor(title, body) {
     this.id = Date.now();
     this.title = title;
     this.body = body;
-    this.star = star || undefined;
+    this.star = false;
```

Comment:
Changed the constructor to set this.star to false by default. We no longer need a star value coming in on line 2. The star value will change by the user's click event later.